workflow/client.go

```diff
@@ -0,0 +1,174 @@
+//go:generate counterfeiter ./ Client
+
+package workflow
+
+import (
+	// "encoding/json"
+	// "errors"
+	// "fmt"
+	"net/http"
+	"net/url"
+	"time"
+
+	"github.com/3dsim/auth0"
+	"github.com/3dsim/workflow-goclient/genclient"
+	"github.com/3dsim/workflow-goclient/genclient/operations"
+	"github.com/3dsim/workflow-goclient/models"
+	"github.com/PuerkitoBio/rehttp"
+	openapiclient "github.com/go-openapi/runtime/client"
+	"github.com/go-openapi/strfmt"
+	// "github.com/go-openapi/swag"
+	log "github.com/inconshreveable/log15"
+)
+
+// Log is a github.com/inconshreveable/log15.Logger.  Log is exposed so that users of this library can set
+// their own log handler.  By default this Log uses the DiscardHandler, which discards log statements.
+// See: https://godoc.org/github.com/inconshreveable/log15#hdr-Library_Use
+//
+// To set a different log handler do something like this:
+//
+// 		Log.SetHandler(log.LvlFilterHandler(log.LvlInfo, log.CallerFileHandler(log.StdoutHandler)))
+var Log = log.New()
+
+func init() {
+	Log.SetHandler(log.DiscardHandler())
+}
+
+const (
+	// WorkflowAPIBasePath is the base path or "slug" for the workflow api
+	WorkflowAPIBasePath = "workflow-api"
```

Comment:
Don't think you need this constant as we now pass in the base path when we create a new client.